packages/components/src/Drawer/Drawer.tsx

```diff
@@ -29,14 +29,14 @@ export function Drawer({
   const drawerClassNames = classnames(styles.container, open && styles.open);
   return (
     <>
-      <div className={drawerClassNames} data-testid="drawer-container">
+      <aside className={drawerClassNames} data-testid="drawer-container">
```

Comment:
@chris-at-jobber my concern here is that `aside` is considered a landmark. Having multiple same landmarks (multiple drawers) on a page will throw an accessibility issue in [AXE](https://www.deque.com/axe/). 

The way around this is to assign an `aria-label` to the `aside`. 

Thoughts on this?

Reply:
Ooh good catch. I think we can assume/hope that you'd only have one Drawer per page, but otherwise I wonder if we could use the value entered as the "title" prop as an aria-label as a failsafe?

Reply:
Yeah I think thats a good idea. do you want to make that change and I will approve this.

Reply:
@eddysims done!